src/zone.ts

```diff
@@ -0,0 +1,80 @@
+import { parseZoneFile, makeZoneFile } from "zone-file";
+import * as fs from 'fs';
+
+export function getDefaultAddress(coin: string, text: any | string): { [key: string]: string } | null {
+    let addresses = getAddresses(coin, text);
+    return (addresses == null ? null : addresses[0]);
+}
+
+export function getAddresses(coin: string, text: any | string): { [key: string]: string }[] | null {
+    // what type to use for text?
```

Comment:
We can assume the zone file will be a string

Reply:
think json as input could be a good option too - we can just convert zone file text to json and work only with the json after